apps/aecore/lib/aecore/peers/worker.ex

```diff
@@ -92,7 +99,31 @@ defmodule Aecore.Peers.Worker do
     {:ok, initial_peers}
   end
 
-  def handle_call({:add_peer, uri}, _from, state) do
+  def handle_call(:is_chain_synced, _from, %{peers: peers} = state) do
+    local_latest_block_height = Chain.latest_block().header.height
+    peer_uris = peers
+      |> Map.values()
+      |> Enum.map(fn(%{uri: uri}) -> uri end)
+    peer_latest_block_heights =
+      Enum.map(peer_uris, fn(uri) ->
+          case Client.get_info(uri) do
+            {:ok, info} ->
+              info.current_block_height
+            :error ->
+              0
+          end
+        end)
+    is_synced =
+      if(Enum.count(peer_uris) == 0) do
```

Comment:
Favor `Enum.empty?` instead of `Enum.count(...) == 0`

`Enum.empty?` works in constant time, whereas `Enum.count` must first count all the elements of a collection, operating in O(N)